[FIX] base,tools: blacklist t-call attribute translation

nle-odoo opened the pull request:
Scenario: enable second language on website, go to /shop, edit
translation => error 500 is shown with error:

   SyntaxError: invalid syntax (<>, line 1)

Cause: since eb6e88a25050fff2bd09317739dd51ba451450df parameters are set
on the t-call tags, but this can conflict with attribute tag that are
translatable. In the case of /shop, there is:

  <t t-call="website_sale.search" placeholder="placeholder"/>

that is transformed to:

  <t t-call="website_sale.search"
     placeholder="<span data-oe-model=...>placeholder</span>"/>

but this is an invalid value, t-call attribute should only be translated
if they have the suffix .translate and should not be auto-translated.

opw-5121074

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/229570.png)](https://mergebot.odoo.com/odoo/odoo/pull/229570)

Gorash reviewed (changes requested):
In this use case, the attribute must be correctly translated/translatable.
```xml
<t t-call="test" placeholder.translate="hello">world</t>
```
Can you add this to your test?

nle-odoo commented:
The fix was preventing translation of: `<t t-call="test" placeholder.translate="hello">world</t>`.

I have modified method `is_translatable_attrib` instead (adding a "node" parameter), and added a test.
